src/base_object.h

```diff
@@ -160,11 +160,7 @@ class BaseObject : public MemoryRetainer {
   // After a successful clone, FinalizeTransferRead() is called on the receiving
   // end, and can read deserialize JS data possibly serialized by a previous
   // FinalizeTransferWrite() call.
-  enum class TransferMode {
-    kUntransferable,
-    kTransferable,
-    kCloneable
-  };
+  enum class TransferMode : uint8_t { kUncloneable, kTransferable, kCloneable };
```

Comment:
Nit: This change is unnecessary. We can revert it to avoid conflicts in the future.